rpcs3/Emu/System.cpp

```diff
@@ -131,31 +132,32 @@ void Emulator::Init(bool add_only)
 
 	g_cfg_defaults = g_cfg.to_string();
 
-	// Reload override configuration set via command line
-	if (!m_config_override_path.empty())
+	// Load config file
+
+	if (m_config_path.find_first_of('/') != umax)
```

Comment:
contains('/') ?

Reply:
This is c++23.

Reply:
I could've sworn we already had contains in c++20 lmao

Reply:
associative containers such as std::map/set has it in c++20, but for string it is c++23.